doc/modules/mod_mam.md

```diff
@@ -31,6 +31,10 @@ For now `odbc` backend has very limited support for this feature, while `cassand
 * **archive_chat_markers** (boolean, default: `false`) - If set to true, XEP-0333 chat markers will be archived. See more details [here](#archiving-chat-markers)
 * **pm** (list | `false`, default: `[]`) - Override options for archivization of one-to-one messages. If the value of this option is `false`, one-to-one message archive is disabled.
 * **muc** (list | `false`, default: `false`) - Override options for archivization of group chat messages. If the value of this option is `false`, group chat message archive is disabled.
+* **extra_lookup_params** (atom, default: `undefined`) - a module implementing `mam_iq` behaviour.
+ If this option has value other then undefined, function `extra_lookup_params/2` from this module will be called when building MAM lookup parameters.
+ This can be used to extend currently supported MAM query fields by some custom field.
```

Comment:
This can be used to extend currently supported MAM query fields by a custom field.